src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      this.removeFromAlive();
+    }
+  }
+
+  removeFromAlive() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
+
+  bite(damage) {
+    this.health -= damage;
+    this.checkHealth();
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden) {
+    super(name, health);
+    this.hidden = hidden || false;
```

Comment:
The constructor of the `Herbivore` class should not accept a `health` parameter, as the health is always initialized to 100 by default according to the task description. Also, the `hidden` property should be initialized to `false` directly without checking for an input parameter.